src/server/config/schema.js

```diff
@@ -30,9 +30,9 @@ module.exports = () => Joi.object({
     exclusive: Joi.boolean().default(false)
   }).default(),
 
-  uuid: Joi.string().guid().default(),
 
   server: Joi.object({
+    uuid: Joi.string().guid().default(),
```

Comment:
Sounds fine, but should probably make sure that @epixa mentions it in the release notes